src/custom/utils/xdai/hack.ts

```diff
@@ -30,3 +30,13 @@ export const switchParamsByNetwork = (chainId?: ChainId) => {
     ETHER.symbol = 'ETH'
   }
 }
+
+export function getChainCurrencySymbols(): { ether?: string; weth?: string } {
```

Comment:
actually this whole function confuses me:

1. `const ether = ETHER.symbol` and below, the `if(ether)` check: `ETHER` comes from the SDK so it's never undefined?
2. why not just return sth like:
```ts
return { ether, weth: ETHER.symbol === 'Ether' ? 'WETH' : 'wxDAI' }
```

Reply:
I've done a change to the implementation, I hope u like more this one :)